packages/_debian-common/changelog

```diff
@@ -5,6 +5,7 @@ libsuperiormysqlpp (0.4.0) UNRELEASED; urgency=medium
   [ Michal Merc ]
   * refactor: Remove szn-debian-wheezy support
   * Extend psReadValues exceptions
+  * feat: Detect lost connection when using query
```

Comment:
It's breaking change only in internal. But as the whole library is header only - even implementation details are actually interface. Im not sure if we need to mark it as API breaking change, but at least note in changelog should be made.

Reply:
If you expand this view, you'll see there is such warning due to other Michal's changes.